src/app.cfg

```diff
@@ -0,0 +1,2 @@
+/home/nlesquoy/ghq/github.com/LaTeX-Project-Manager/settings/config.json
```

Comment:
same remark as above regarding the hardcoded paths :thinking: 
you could parse `$HOME` inside the code :+1: